src/dispatch/report/flows.py

```diff
@@ -80,6 +78,11 @@ def create_executive_report(
     user_id: str, user_email: str, incident_id: int, action: dict, db_session=None
 ):
     """Creates an executive report."""
+    report_template = document_service.get_executive_report_template(db_session=db_session)
+
+    if not report_template:
```

Comment:
I don't think we have a good story around reporting errors back to a slack user. cc @mvilanova for ideas.

Reply:
You can call the following function to provide feedback to the user: https://github.com/Netflix/dispatch/blob/develop/src/dispatch/conversation/messaging.py#L5